### ISSUE TITLE
problems with validation set on a similar project

### ISSUE THREAD

**Issue by radiodee1** [dandelin/Dynamic-memory-networks-plus-Pytorch#3]
Hi. I'm pretty much a newbie at neural networks. This is a question I asked on the yerevann blog site but I never got an answer. I'm using your code as something of a guide along with the yerevann code. I'm programming in Pytorch. (I also use part of the yerevann data loader. That code is properly attributed to them.) I'm trying to implement a similar DMN for the babi set. My code is here: https://github.com/radiodee1/awesome-chatbot . Here's a question I have.

I split the training and testing data into parts for training, testing, and validation. I use 1000 parts for training, 500 parts for testing and 500 parts for validation. I run into a problem. I can train successfully but when I go to the validation step I never get a score above 50% accuracy. My question to you is what part of the program would be responsible for this kind of behavior? In other words, can you tell me why I'm always getting 50% ?? Thanks for your time. I understand this blog is pretty old. I'm limiting my experiments to the first babi test. I thought I had an answer to this question and then it popped up again and I cannot get rid of it.

thanks again.
